### PR TITLE
Complete the mappings.

### PR DESCRIPTION
**Pull request by awong-dev**
With this, the mappings should be done.  Next is to verify with ES and
do bug fixing + tests.

### CODE REVIEW

File: src/common/veteran.js [hunk 1]
```diff
@@ -67,13 +67,13 @@ const blankVeteran = {
...
-    day: makeField(''),  // TODO(awong): Ignored by ES System
-    year: makeField('')  // TODO(awong): Ignored by ES System
+    month: makeField(''),
+    day: makeField(''),
+    year: makeField('')
   },
   sameAddress: makeField(''),  // TODO(awong): Not sure how to handle the mapping.
-  cohabitedLastYear: makeField(''),  // TODO(awong): This should be scoped to spouse.
-  provideSupportLastYear: makeField(''),  // TODO(awong): This should be scoped to spouse.
+  cohabitedLastYear: makeField(''),  // TODO(awong): This name should be scoped to spouse.
```
Comment (robbiethegeek) [inline src/common/veteran.js]: What is still left on this set of TODOs?

**Comment by robbiethegeek** [comment]
One minor question that is about a TODO so I am LGTM and merging this in.  @awong-dev take a look at the question about the TODO and if not captured as an issue add to the backlog?

File: src/common/veteran.js (hunk 1 above)
Comment (awong-dev) [inline src/common/veteran.js]: Mostly I want to shift it into a `spouse {}` object.  But I think with the next PR, this should move into the API layer.

Comment (awong-dev) [inline src/common/veteran.js]: I consider this part of validation + testing as I think unittesting will make us really want to refactor the object structure a little.

Comment (robbiethegeek) [inline src/common/veteran.js]: makes sense was just trying to grok the todo :) thanks!

**Pull request merged**